cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec

```diff
@@ -99,6 +99,9 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Thu Feb 3 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.16.0
+- [cygnus-ngsi][KafkaSink] Upgrade libthrift dependency from 0.12.0 to 0.14.1 due to github vulnerability report
```

Comment:
```suggestion
- [cygnus-ngsi][NGSIHandler] Check TextUnrestricted type to escape character ' as '' (#2125)
- [cygnus-ngsi][KafkaSink] Upgrade libthrift dependency from 0.12.0 to 0.14.1 due to github vulnerability report
```

Moving to [cygnus-ngsi] (as in PR #2128 it seems all modifications are under cygnus-ngsi/ directory)